contrib/IECoreUSD/src/IECoreUSD/USDScene.cpp

```diff
@@ -1020,6 +1019,65 @@ IECoreScene::CurvesPrimitivePtr convertPrimitive( pxr::UsdGeomCurves curves, pxr
 	return newCurves;
 }
 
+IECore::ConstObjectPtr convertCamera( pxr::UsdGeomCamera camera, pxr::UsdTimeCode time)
+{
+    float horizontalAper=20.955,  horizontalOff = 0.0,  verticalAper =15.2908, verticalOff = 0.0;
+    camera.GetHorizontalApertureAttr().Get(&horizontalAper, time);
+    camera.GetVerticalApertureAttr().Get(&verticalAper, time);
+    camera.GetHorizontalApertureOffsetAttr().Get(&horizontalOff, time);
+    camera.GetVerticalApertureOffsetAttr().Get(&verticalOff, time);
+
+    IECoreScene::CameraPtr result = new IECoreScene::Camera;
+    pxr::TfToken  projToken;
+    camera.GetProjectionAttr().Get(&projToken);
+
+    if(projToken == pxr::UsdGeomTokens->perspective)
+    {
+        result->setProjection("perspective");
+
+        // We store focalLength and aperture in arbitary units.  USD uses tenths
+        // of scene units
+        float scale = 10.0f * result->getFocalLengthWorldScale();
+
+        float focalLengthVal = 1.0;
+        camera.GetFocalLengthAttr().Get( &focalLengthVal, time );
+        result->setFocalLength(focalLengthVal / scale);
+        result->setAperture(Imath::V2f(horizontalAper / scale, verticalAper / scale));
+        result->setApertureOffset(Imath::V2f(horizontalOff / scale,  verticalOff / scale));
```

Comment:
I don't _think_ we need to factor in `scale` here. I believe it cancels out as long as they're in the same units. The only time the actual dimensions become meaningful is in relation to `setFocusDistance` and the resulting DoF blur. @danieldresser-ie may be able to confirm.